bindings/python/pire.swg

```diff
@@ -0,0 +1,17 @@
+%module pire
+
+%include <exception.i>
+
+%exception {
+    try {
+        $action
+    }
+    SWIG_CATCH_STDEXCEPT
+    catch (...) {
+        SWIG_exception(SWIG_UnknownError, "C++ anonymous exception");
```

Comment:
This is a snippet from arcadia. The exception is anonymous in sense that it does not inherit from std::exception.